delta-lake/common/src/main/databricks/scala/com/databricks/sql/transaction/tahoe/rapids/GpuWriteIntoDelta.scala

```diff
@@ -22,15 +22,15 @@
 package com.databricks.sql.transaction.tahoe.rapids
 
 import com.databricks.sql.transaction.tahoe.{DeltaOperations, OptimisticTransaction}
-import com.databricks.sql.transaction.tahoe.commands.WriteIntoDelta
+import com.databricks.sql.transaction.tahoe.commands.WriteIntoDeltaEdge
 
 import org.apache.spark.sql._
 import org.apache.spark.sql.execution.command.LeafRunnableCommand
 
 /** GPU version of Delta Lake's WriteIntoDelta. */
 case class GpuWriteIntoDelta(
     gpuDeltaLog: GpuDeltaLog,
-    cpuWrite: WriteIntoDelta)
+    cpuWrite: WriteIntoDeltaEdge)
```

Comment:
Does this work for other versions of Deltalake on databricks?

Reply:
Databricks build is definitely broken